src/Chapter3.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE DerivingStrategies #-}
```

Comment:
Nice set of extensions! :earth_africa: 
These two extensions are considered safe and helpful, and it's generally a good idea to enable them by default :+1:

Reply:
Thanks! So far, I always set the extensions explicitly in my little toy projects for learning purposes, but I think it's time to step it up and use default extensions :smiley: